### PR TITLE
few cmds help cleanup

### PR DESCRIPTION
**Pull request by ajmohan**
**enc**:
- typo in -base64 option
- missing help opt text

**ocsp/req/rsautl/s_client**:
  - missing help opt text

### CODE REVIEW

**Comment by ajmohan** [comment]
squashed into one commit. Please have a look.

**Comment by ajmohan** [comment]
@mattcaswell, @richsalz : can you please review the changes.

**Comment by richsalz** [comment]
Yes, a little patience please. :)

**Comment by ajmohan** [comment]
sure. Thanks

File: apps/enc.c [hunk 1]
```diff
@@ -97,14 +97,17 @@ OPTIONS enc_options[] = {
...
-    {"v", OPT_V, '-'},
+    {"v", OPT_V, '-', "Verbose output"},
     {"nopad", OPT_NOPAD, '-', "Disable standard block padding"},
-    {"salt", OPT_SALT, '-'},
-    {"nosalt", OPT_NOSALT, '-'},
-    {"debug", OPT_DEBUG, '-'},
-    {"A", OPT_UPPER_A, '-'},
-    {"a", OPT_A, '-', "base64 encode/decode, depending on encryption flag"},
-    {"base64", OPT_A, '-', "Base64 output as a single line"},
+    {"salt", OPT_SALT, '-',
```
Comment (richsalz) [inline apps/enc.c]: put on one line.

File: apps/enc.c [hunk 2]
```diff
@@ -97,14 +97,17 @@ OPTIONS enc_options[] = {
...
+     "Use salt in the KDF (default)"},
+    {"nosalt", OPT_NOSALT, '-',
```
Comment (richsalz) [inline apps/enc.c]: put on one line.

File: apps/enc.c [hunk 3]
```diff
@@ -97,14 +97,17 @@ OPTIONS enc_options[] = {
...
+     "Do not use salt in the KDF"},
+    {"debug", OPT_DEBUG, '-', "Print debug info"},
+    {"a", OPT_A, '-', "Base64 encode/decode, depending on encryption flag"},
+    {"base64", OPT_A, '-', "same as option -a"},
```
Comment (richsalz) [inline apps/enc.c]: Uppercase same

File: apps/req.c [hunk 4]
```diff
@@ -143,12 +143,13 @@ OPTIONS req_options[] = {
     {"config", OPT_CONFIG, '<', "Request template file"},
     {"keyout", OPT_KEYOUT, '>', "File to send the key to"},
     {"passin", OPT_PASSIN, 's', "Private key password source"},
-    {"passout", OPT_PASSOUT, 's'},
+    {"passout", OPT_PASSOUT, 's',
```
Comment (richsalz) [inline apps/req.c]: put on one line?

File: apps/rsautl.c [hunk 5]
```diff
@@ -98,10 +98,12 @@ OPTIONS rsautl_options[] = {
     {"oaep", OPT_OAEP, '-', "Use PKCS#1 OAEP"},
     {"sign", OPT_SIGN, '-', "Sign with private key"},
     {"verify", OPT_VERIFY, '-', "Verify with public key"},
-    {"asn1parse", OPT_ASN1PARSE, '-'},
+    {"asn1parse", OPT_ASN1PARSE, '-',
+     "asn1parse the output data, useful with -verify"},
```
Comment (richsalz) [inline apps/rsautl.c]: change comma to semicolon.
Maybe "Run output through asn1parse; useful with -verify" as the text?

File: apps/rsautl.c [hunk 6]
```diff
@@ -98,10 +98,12 @@ OPTIONS rsautl_options[] = {
...
     {"hexdump", OPT_HEXDUMP, '-', "Hex dump output"},
     {"x931", OPT_X931, '-', "Use ANSI X9.31 padding"},
-    {"rev", OPT_REV, '-'},
+    {"rev", OPT_REV, '-',
```
Comment (richsalz) [inline apps/rsautl.c]: put on one line.

File: apps/s_client.c [hunk 7]
```diff
@@ -715,21 +716,31 @@ OPTIONS s_client_options[] = {
...
+    {"brief", OPT_BRIEF, '-',
+     "Restrict output to brief summary of connection parameters"},
+    {"prexit", OPT_PREXIT, '-',
+     "Print session information when the program exits"},
+    {"security_debug", OPT_SECURITY_DEBUG, '-',
+     "Enable security debug messages"},
+    {"security_debug_verbose", OPT_SECURITY_DEBUG_VERBOSE, '-',
+     "Output more security debug output"},
+    {"cert_chain", OPT_CERT_CHAIN, '<',
+     "Certificate chain file in PEM format"},
```
Comment (richsalz) [inline apps/s_client.c]: parens around "(in PEM format)" ?

File: apps/s_client.c [hunk 8]
```diff
@@ -741,7 +752,7 @@ OPTIONS s_client_options[] = {
     {"alpn", OPT_ALPN, 's',
      "Enable ALPN extension, considering named protocols supported (comma-separated list)"},
     {"async", OPT_ASYNC, '-', "Support asynchronous operation"},
-    {"ssl_config", OPT_SSL_CONFIG, 's'},
+    {"ssl_config", OPT_SSL_CONFIG, 's', "Use configuration file"},
```
Comment (richsalz) [inline apps/s_client.c]: "Use specified configuration file"

File: apps/s_client.c [hunk 9]
```diff
@@ -758,18 +769,19 @@ OPTIONS s_client_options[] = {
     {"tls1_2", OPT_TLS1_2, '-', "Just use TLSv1.2"},
 #endif
 #ifndef OPENSSL_NO_DTLS
-    {"dtls", OPT_DTLS, '-'},
-    {"timeout", OPT_TIMEOUT, '-'},
+    {"dtls", OPT_DTLS, '-', "Use any DTLS"},
```
Comment (richsalz) [inline apps/s_client.c]: "Use any version of DTLS"

**Comment by richsalz** [comment]
This is nice.  Great to have good help output.  A few nits, then update the commit and re-push.

**Comment by ajmohan** [comment]
Done. Please have a look. Thanks

**Comment by richsalz** [comment]
+1  I'll merge after someone else reviews.  THANKS!

**Comment by t-j-h** [comment]
@richsalz  +1

**Comment by richsalz** [comment]
pushed in 6c335b0 thanks you!

**Pull request closed without merging**